0007/README.md

```diff
@@ -0,0 +1,221 @@
+of Steering Committee members. This gives us a natural point for reflection on
+the role and operation of the Steering Committee. Several project contributors
+have raised justifiable concerns about the lack of transparency of the
+committee, the ambiguity in how responsibilities differ for the role of a
+Steering Committee member vs. Maintainers, and what the relationship is with
+SLSA’s parent [Supply Chain Integrity Working Group (SCI WG)][SCI WG].
+
+### Historical context
+
+The SLSA Steering Committee was formed during the early days of both SLSA and
+the OSSF to provide a diverse set of inputs and oversight to the specification.
+Since the committee's original formation, the OpenSSF has developed more
+[structure][OSSF structure] and [processes][OSSF processes] which could
+support the SLSA project.
+
+## Prior art
+
+Many governance models exist for steering or umbrella projects, i.e.; the
+current [SLSA Steering Committee], [Kubernetes Steering Council],
+numerous [other models] within the CNCF.
+
+## Goals
+
+[goals]: #goals
+
+*   High-level vision/strategy for the SLSA project should be shaped through
+    diverse inputs, including from voices outside of the SLSA project’s active
+    contributors.
+*   Transparency at all levels of SLSA governance, with clearly defined roles
+    and responsibilities.
+*   Greater recognition of existing, and future, active contributors.
+*   Strong sense of ownership for active contributors.
+*   Diverse affiliation of community leadership, such that a single entity
+    cannot unduly influence the project.
+*   Lower cost of operations for SLSA governance. The SLSA organization is
+    small, so a heavyweight process is not worth the effort. In addition, SLSA
+    can lean on the [established organizational support][OSSF structure] in the
+    OpenSSF if needed.
+
+
+## Proposed solution
+
+Below is a summary of the proposed changes, highlighting the important
+differences in **bold**. The purpose of this proposal is to get general
+agreement on the idea; exact wording will be worked out in the implementation
+PR. See [current governance doc] for reference, including the list of example
+duties for each role.
+
+### Replace “terms” / “seats” with a lightweight re-application process
+
+*   SC **no longer has 7 fixed seats**, but instead has a minimum of 3 and
+    ideally about 5.
+    *   Same guidelines for Maintainers, without the hard minimum.
+*   Individuals **self-nominate** to become a Maintainer or SC Member by
+    submitting a brief (1-2 paragraph) application consisting of:
+    *   **Evidence** of performing some of the role's duties over the recent
+        past.
+        *   For SC Member applications, this should include evidence of being
+            well-versed in SLSA.
+    *   **Statement of intent** to continue performing duties over the next
+        year.
+    *   **Affiliation** / employer.
+*   **SC Members decide** if the individual meets the desirable properties
```

Comment:
Shouldn't we allow the community to vote as well for the SC Members or maintainers seat?  SC can have the majority vote, but if you don't have community consensus, this leads to potentially bias.

Reply:
We have been trying to avoid a formal voting process, as they can be onerous for the people running them. That also helps us avoid the question of how to identify elligible voters.

I hope that the public nomination process will help keep people honest. If we are to use GitHub Issues for nominations, that would allow folks to express support (though admittedly in a non-anonymous way).

Reply:
IMO voting is overkill for such a small community. If the community grows 10x in size, then maybe we reconsider?

Reply:
The biggest challenge in holding such a vote is that it raises the question of who gets to vote. For this reason, I think it is better to keep the decision to the current maintainers and SC members even though it is prone to conflicts of interest.

Reply:
I think it's important to have a mechanism for making hard decisions (voting), and then do your best to avoid using it (e.g., by determining why there are differences and working out compromises that address all major concerns).